graphics/font.h

```diff
@@ -104,6 +104,24 @@ class Font {
 	 */
 	virtual int getFontAscent() const;
 
+	/**
+	 * Return the descent of the font.
+	 *
+	 * @return Font descent in pixels. If it is unknown
```

Comment:
```suggestion
	 * @return Font descent in pixels. If it is unknown,
```

Reply:
I'm leaving this one for consistency because it's a copy of existing sentences on other functions in this font header. You can have all your superfluous SCI commas though ;)